CLI/add_feature_extractor.py

```diff
@@ -117,10 +122,13 @@ def parser_function() -> argparse.ArgumentParser:
             + gv.get_time_pid_random_string()
             + ".rawres"
         )
-        command_line = [extractor_target_path / gv.sparkle_run_default_wrapper,
-                        f"{extractor_target_path}/",
-                        str(instance_path),
-                        result_path]
+        command_line = [
+            "python3", str(extractor_target_path / gv.sparkle_extractor_wrapper),
```

Comment:
There is a gv variable for the python3, and we don't need to stringify Path objects before passing them to subprocess.run (Do check if this command_line list is used somewhere else though before removing the stringification)